docs/upgrade/0.8.0_to_0.9.0.rst

```diff
@@ -0,0 +1,123 @@
+Upgrade from 0.8.0 to 0.9.0
+===========================
+
+Updating the Tails Workstations
+-------------------------------
+
+We recommend that you update all Tails drives to version 3.9, which was released
+concurrently with SecureDrop 0.9.0 on September 5, 2018. Follow the
+graphical prompts on your workstations to perform this upgrade.
+
+For the *Journalist Workstations* and the *Admin Workstation*, the graphical
+SecureDrop updater will also prompt you to update the SecureDrop code on your
+workstations. The updater was introduced in SecureDrop 0.7.0. It looks like
+this:
+
+.. |SecureDrop updater| image:: ../images/0.6.x_to_0.7/securedrop-updater.png
+
+|SecureDrop updater|
+
+If the updater does not appear, you may not have updated your workstation code
+since the release of SecureDrop 0.7.0 in May 2018. You can manually update your
+workstation by issuing the following commands on each workstation:
+
+.. code:: sh
+
+   cd ~/Persistent/securedrop
+   git fetch --tags
+   gpg --recv-key "2224 5C81 E3BA EB41 38B3 6061 310F 5612 00F4 AD77"
+   git tag -v 0.9.0
+
+The output should include the following two lines:
+
+.. code:: sh
+
+   gpg:                using RSA key 22245C81E3BAEB4138B36061310F561200F4AD77
+   gpg: Good signature from "SecureDrop Release Signing Key"
+
+Please verify that each character of the fingerprint above matches what you see
+on the screen of your workstation. If it does, you can check out the new
+release:
+
+.. code:: sh
+
+  git checkout 0.9.0
+
+Please verify that the output of this command does not contain
+the text "warning: refname '0.9.0' is ambiguous".
+
+.. important:: If you do see the warning "refname '0.9.0' is ambiguous" in the
+  output, we recommend that you contact us immediately at securedrop@freedom.press
+  (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__).
+
+Finally, run the following command:
+
+.. code:: sh
+
+  ./securedrop-admin setup
+
+Please note that this only updates the SecureDrop code on the workstation.
+Tails upgrades still have to be performed separately.
+
+New Dialogs in Tails 3.9
```

Comment:
thought: we should add a note about the database migrations: "Part of the release includes a significant database migration that can take some time, especially if you are not using an SSD or if you have a large number of sources and submissions. You should not be concerned as this is normal behavior during the upgrade, and the source/journalist interfaces will become available again once the upgrade has completed."